docs/meshstack.aws.aad-as-idp.md

```diff
@@ -0,0 +1,80 @@
+---
+id: meshstack.aws.aad-as-idp
+title: AAD as Identity Provider
+---
+## Configure AAD as IdP for AWS SSO
+
+You can trigger everything that is needed for using AAD as an IdP for AWS SSO from within Azure.
+
+### Create an Enterprise Registration
+
+Search for `aws` and choose `AWS Single Sign-on` as the enterprise registration type
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_01.png)
+
+Follow the steps in AAD under Getting Started (Pick SAML for the single sign on.)
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_02.png)
+
+### Configure AWS for AAD integration
+
+Choose `AWS Single Sign-on`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_03.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_03.png)
+
+
+Configure the identity source by clicking on `Choose your identity source`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_04.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_04.png)
+
+We have to change the `Identity source`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_05.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_05.png)
+
+Select `External identity provider`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_06.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_06.png)
+
+### Metadata
+
+Download `AWS SSO SAML metadata` from AWS and then you should upload it into `Single sing-on` page on Azure with the `Upload metadata file` button.
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_07.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_07.png)
+
+Also check the following image to download `IdP SAML metadata` file from Azure, you can find it in the same page as `Federation Metadata XML`
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_08.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_08.png)
+
+After downloading the `Federation Metadata XML` from Azure you have to upload this file into your AWS SSO Identity Source under `IdP SAML metadata`
+
+## Provisioning
+
+Before the automatic provisioning is enabled, you can click `enable automatic provisioning` on Azure to get the Tenant URL and Access Token to configure SCIM for provisioning on AAD side.
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_09.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_09.png)
+
+### euid not usable
+
+In AWS SSO lookup of users is only possible via userName, not via email (see [https://docs.aws.amazon.com/singlesignon/latest/developerguide/listusers.html](https://docs.aws.amazon.com/singlesignon/latest/developerguide/listusers.html)). Sadly we can't set userName in AWS SSO to the user's email address as this mapping cannot be changed for the automatic SCIM provisioning in AAD.
+
+![assets/aad_as_idp_for_aws/aad_as_idp_for_aws_10.png](assets/aad_as_idp_for_aws/aad_as_idp_for_aws_10.png)
+
+You can also disable the Group sync here, as usually only groups managed by meshStack will be used and they will be created by meshStack anyway.
```

Comment:
q: This means it's optional, right?

Reply:
Disabling the group sync is the default that should be done on AAD side, as we take care of managing groups.

Reply:
So should it stay that way?

Reply:
Changed